README.md

```diff
@@ -83,3 +84,13 @@ class Command(BaseCommand):
                         old_cache.delete(actual_key)
 
 ```
+
+# Scheduler related usage
+You can use some extra functionality provided by this plugin, if secure cache settings are not added in the `default` settings, please make sure that you specify your secure cache settings name by setting `DJANGO_REDIS_SECURE_CACHE_NAME` settings. To enable these feature, this simply by adding `@secure_redis.rq_job` decorator on top of your task. By doing this you can get use of:
+
+1. `delay` method, so you can use it like `my_job.delay()`. This method is same as `django_rq.job.delay`
+2. `enqueue_at` method, called as `my_job.enqueue_at()`. This is an exposed method of `django_rq.Scheduler.enqueue_at`
+3. `schedule_once` method, called as `my_job.schedule_once()`. Same as `django_rq.Scheduler.schedule` but will check if method already exists so it will not added twice.
```

Comment:
3. A `schedule_once` method, which can be used when calling the task method (ex: `my_job.schedule_once()`). This method has the same functionality as `django_rq.Scheduler.schedule`, but will check if the method already exists and will not add it to the scheduler a second time.